common/src/version.rs

```diff
@@ -23,6 +23,8 @@ pub fn get_shuttle_service_from_user_crate<P: AsRef<Path>>(manifest_path: P) ->
             "--format",
             "{p}",
         ])
+        .arg("--color")
```

Comment:
Hmm... This turns on colouring when we check the versions. It still should not affect `cargo shuttle run` when you try it locally right?

Reply:
You told me to make it work in the compile function ig, so it does

Reply:
I'm confused, how is this the compile function I linked to? This is the `get_shuttle_service_from_user_crate` function

Reply:
oh god, I got mixed up, sorry